File: Daily Test/Python/Compress & Sort - Integers 29.10.2025.py
n, l, l1 = int(input()), input().split(), []
for i in l:
    l2 = [int(i[:j] + i[j+1:]) for j in range(len(i))]
    l1.append(min(l2))
l1.sort()
print(*l1)

File: Daily Test/Java/Compress & Sort - Integers 29.10.2025.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        List<Integer> l1 = new ArrayList<>();

        for (int i = 0; i < n; i++) {
            String s = sc.next();
            List<Integer> l2 = new ArrayList<>();
            for (int j = 0; j < s.length(); j++) {
                String temp = s.substring(0, j) + s.substring(j + 1);
                l2.add(Integer.parseInt(temp));
            }
            l1.add(Collections.min(l2));
        }

        Collections.sort(l1);
        for (int val : l1)
            System.out.print(val + " ");
        System.out.println();
        sc.close();
    }
}